EFCoreAutoMigrator/AutoMigrator.cs

```diff
@@ -110,6 +115,14 @@ public void Migrate(bool forceMigration, MigrationModelHashStorageMode mode, Act
             }
         }
 
+        private void ExecuteBeforeMigration(Action preMigration)
+        {
+            WriteMessage("Executing database pre process function...");
+            preMigration.Invoke();
+            WriteMessage("Migration sequence has been completed.");
```

Comment:
Can we change this to "Migration sequence has been completed." ?

Reply:
Sure